example-app/app/Models/Airline.php

```diff
@@ -9,9 +9,12 @@ class Airline extends Model
 {
     use HasFactory;
 
+    protected $fillable = ['name', 'description'];
```

Comment:
You can do the opposite, declare a guarded, so every attribute but the guarded can be mass asigned
```suggestion
    protected $guarded = ['id'];
```